protobuf.c

```diff
@@ -382,9 +382,9 @@ PHP_METHOD(ProtobufMessage, parseFromString)
 
 	if (pb_get_field_descriptors(getThis(), &field_descriptors))
 		return;
-
-	if ((values = pb_get_values(getThis())) == NULL)
-		goto fail0;
+	values = pb_get_values(getThis());
```

Comment:
This is   another bug, it will be failed if we define some protocol with  an empty structure/message like below : 
`
optional QueryDailyJob          QueryDailyJob           = 214;
message QueryDailyJob {
}
`
PHP will skip the QueryDailyJob, and  all structure like that will be skipped. it is not the same with other languages like java, c#, c++ and OC etc.

Reply:
I have changed the commit's comment

Reply:
I was a bit surprised a _values_ field is not defined for empty messages. This change seems valid now  :)

You left commented out two lines of code. Can you remove them?

Reply:
:) , some guy use a empty structure as a signal. I have removed the commented lines, Please review. thank you for pointing that out , I have learned a lot from you about the coding standards.